charts/fluentd/templates/daemonset.yaml

```diff
@@ -11,8 +11,8 @@ spec:
       {{- include "fluentd.selectorLabels" . | nindent 6 }}
   template:
     metadata:
-      {{- with .Values.podAnnotations }}
       annotations:
```

Comment:
this will render `annotations:` which is probably not desired, we can move make this change if at some point we hardcode some annotations here.